test/lit.cfg.py

```diff
@@ -57,14 +57,11 @@
 
 if config.spirv_tools_have_spirv_as:
     llvm_config.add_tool_substitutions(['spirv-as'], [config.spirv_tools_bin_dir])
-    config.available_features.add('spirv-as')
     using_spirv_tools = True
 
 if config.spirv_tools_have_spirv_val:
     llvm_config.add_tool_substitutions(['spirv-val'], [config.spirv_tools_bin_dir])
     using_spirv_tools = True
-else:
-    config.substitutions.append(('spirv-val', ':'))
 
 if using_spirv_tools:
```

Comment:
Should also we remove `using_spirv_tools` and the lines above that set it, and unindent the following lines?